lib/improver/tests/test_ensemble_copula_coupling_EnsembleCopulaCouplingUtilities.py

```diff
@@ -41,10 +41,161 @@
 import numpy as np
 
 from improver.ensemble_copula_coupling.ensemble_copula_coupling_utilities \
-    import create_percentiles, create_cube_with_percentiles
+    import (create_percentiles, create_cube_with_percentiles,
+            insert_lower_and_upper_endpoint_to_1d_array,
+            concatenate_2d_array_with_2d_array_endpoints,
+            get_bounds_of_distribution,
+            reshape_array_to_have_probabilistic_dimension_at_the_front)
+from improver.ensemble_copula_coupling.ensemble_copula_coupling_constants \
+    import bounds_for_ecdf
 from improver.tests.helper_functions_ensemble_calibration import (
     set_up_temperature_cube, set_up_spot_temperature_cube,
-    _add_forecast_reference_time_and_forecast_period)
+    _add_forecast_reference_time_and_forecast_period,
+    set_up_probability_above_threshold_temperature_cube)
+
+
+class Test_reshape_array_to_have_probabilistic_dimension_at_the_front(
```

Comment:
Minor thing, but it might help to have these tests in the same order as the in the utilities.py